backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+    async index(request, response) {
+        const user = await connection('user').select('*');
+    
+        return response.json(user);
+    },
+
+    async create(request, response){
+        const { username, password, email, active } = request.body;
+
+        try { 
+            await connection('user').insert({
+                username,
+                password,
+                email,
+                active
+            })
+        } catch (exception) {
+            console.log('Exception: ' + exception);
+            return response.status(400).send();
+        }
+
+        return response.status(200).send();
+    },
+
+    async delete(request, response){
```

Comment:
eslint: [space-before-function-paren](http://eslint.org/docs/rules/space-before-function-paren) (and after too)

Reply:
Done!